slack/client.go

```diff
@@ -414,6 +415,52 @@ func (c *slackClient) loadFromStorage() error {
 	return nil
 }
 
+// Broadcast sends message, and returns the number of
+// processed bots, conversations, errors and error occurred
+func (c *slackClient) Broadcast(msg interface{}, cond flamingo.Condition) (uint64, uint64, uint64, error) {
+	var err error
+	conversationsCount := uint64(0)
+	clientBotsCount := uint64(0)
+	errorsFound := uint64(0)
+
+	for _, clientBot := range c.bots {
```

Comment:
i think i'd rather have the errors printed in the log and ignored than all the condition logic